hat/test/org/hatdex/hat/she/functions/DataFeedDirectMapperSpec.scala

```diff
@@ -77,7 +77,7 @@ class DataFeedDirectMapperSpec extends PlaySpecification with Mockito with DataF
       transformed.source must be equalTo "google"
       transformed.types must contain("event")
       transformed.title.get.text must contain("MadHATTERs Tea Party: The Boston Party")
-      transformed.title.get.subtitle.get must contain("12 December 18:30 - 22:30 EST")
+      transformed.title.get.subtitle.get must contain("12 December 18:30 - 22:30 America/New_York")
```

Comment:
Just a comment. Really don't like our tests that involve time...

Reply:
Agreed